### ISSUE TITLE
Tetris DS WiFi menu returns to incorrect game

### ISSUE THREAD

**Issue by T0biasCZe** [YANBForwarder/YANBF#32]
In Tetris DS, when you go into WiFi options, and then go back, it starts last player game through Twilight Menu++ instead of Tetris DS. When the game is launched through Twilight Menu ++ it doesn't do this

**Comment by T0biasCZe** [comment]
https://imgur.com/a/ftWcJQ8

**Comment by lifehackerhansol** [comment]
Can you please check if the DSiWare based forwarder also has this issue?

https://wiki.ds-homebrew.com/ds-index/forwarders.html

**Comment by T0biasCZe** [comment]
oof, that one does it too... (But it doesn't take minute to load the WiFi menu)
So issue is somewhere else

**Comment by T0biasCZe** [comment]
Also, found that it's not the last game that was launched through Twilight menu, it launches sonic rush because it's the first DsiWare that's installed on the 3DS

**Comment by lifehackerhansol** [comment]
I'll test this myself, and if it reproduces I'll bring it up to Robz, who develops the other half of the templates.

Stay tuned.

**Comment by T0biasCZe** [comment]
Well I tried uninstalling sonic rush and when going from the WiFi options, the 3ds crashed. So I uninstalled both the forwarders, recreated them and reinstalled. It returned to the game without problems from th Wifi settings. So I also tried installing another dsiware gam, and it also returned from the WiFi config without issues. To see there isn't issue anywhere else, I uninstalled all the forwarders, installed back Sonic Rush, then installed the tetris forwarders, and it still returns from the WiFi menu correctly... So there was issue somewhere. But it doesn't happen anymore after the reinstall
